proposals/required-members.md

```diff
@@ -263,42 +263,131 @@ init_argument_initializer
 A type with a parameterless constructor that advertises a _contract_ is not allowed to be substituted for a type parameter constrained to `new()`, as there is no way
 for the generic instantiation to ensure that the requirements are satisfied.
 
+### Overriding, Hiding, and Inheriting
+
+It is an error to mark a member required if the member is not at least as visible as the containing member's type. This means the following cases are not allowed:
```

Comment:
Consider using the phrase "less accessible than" instead of "not at least as visible as".

Consider using "containing type" instead of "containing member's type". If the thing contains required members, it is always a type, right?